src/VisualStudio/LiveShare/Impl/LiveShareInitializeHandler.cs

```diff
@@ -22,7 +22,7 @@ internal class LiveShareInitializeHandler : ILspRequestHandler<InitializeParams,
                 CodeActionProvider = true,
                 ExecuteCommandProvider = new ExecuteCommandOptions(),
                 ReferencesProvider = true,
-                RenameProvider = true,
+                RenameProvider = false,
```

Comment:
why?

Reply:
This is the liveshare LSP server - since we're moving this from being handled by liveshare to the VS LSP client, the liveshare LSP server should not advertise it supports rename.

Reply:
👍